pkg/solana/txm/txm.go

```diff
@@ -148,9 +148,12 @@ func (txm *Txm) sendWithRetry(chanCtx context.Context, baseTx solanaGo.Transacti
 	// https://github.com/gagliardetto/solana-go/blob/main/transaction.go#L252
 	key := baseTx.Message.AccountKeys[0].String()
 
+	// only calculate base price once
```

Comment:
if bumping occurs after a new block is produced, wouldn't re-calculating the base be helpful?

Reply:
that's true - but there is also the possibility that the base value decreases and then the bumped value could be lower than the previous value.

only calculating the base once guarantees that the fee will always increase